src/seedu/addressbook/Main.java

```diff
@@ -107,6 +108,7 @@ private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
+            checkIfStorageFileExists();
```

Comment:
You shouldn't checked the file here, this doesn't satisfy SLAP. Better to check inside `save` method.